ozpricechecker/pricefinderapp/models.py

```diff
@@ -1,3 +1,140 @@
-# from django.db import models
+# from enum import Enum
+from django.db import models
+from django.contrib.auth.models import User
+
 
 # Create your models here.
+class Currency(models.Model):
+    name = models.CharField(max_length=30)
+
+    def __str__(self):
+        return self.name
+
+    class Meta:
+        db_table = "currency"
+
+
+class Store(models.Model):
+    name = models.CharField(max_length=30)
+    prod_base_url = models.CharField(max_length=250)
+    dynamic_page = models.BooleanField(default=False)
+    currency = models.ForeignKey(Currency, on_delete=models.CASCADE)
+
+    def __str__(self):
+        return self.name
+
+    class Meta:
+        db_table = "Store"
+
+
+class StoreProduct(models.Model):
+    store_id = models.ForeignKey(Store, on_delete=models.CASCADE)
+    prod_url = models.CharField(max_length=250)
+    name = models.CharField(max_length=30)
+
+    def __str__(self):
+        return self.name
+
+    class Meta:
+        db_table = "StoreProduct"
+
+
+class ProductPrice(models.Model):
+    store_prod_id = models.ForeignKey(StoreProduct, on_delete=models.CASCADE)
+    date_time = models.DateTimeField(auto_now=True)
+    price = models.IntegerField()
+
+    # def __str__(self):
+    #     return self.price
+
+    class Meta:
+        db_table = "ProductPrice"
+
+
+class UserProduct(models.Model):
+    store_prod_id = models.ForeignKey(StoreProduct, on_delete=models.CASCADE)
+    user_id = models.ForeignKey(User, on_delete=models.CASCADE)
+    threshhold = models.IntegerField()
+
+    class Meta:
+        db_table = "Userproduct"
```

Comment:
A 3rd variation of table naming style. Let's just use default Django behviour without specifying the name via Meta.